src/main/resources/templates/fragments/topnav.html

```diff
@@ -5,14 +5,17 @@
             <div class="collapse navbar-collapse" id="navbarNav">
                 <ul class="navbar-nav">
                     <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="#">Home</a>
+                        <a class="nav-link active" aria-current="page" href="/">Home</a>
                     </li>
                     <li class="nav-item">
                         <a class="nav-link" href="#">Metrics</a>
                     </li>
                     <li class="nav-item">
                         <a class="nav-link" th:href="@{/login}">Login</a>
                     </li>
+                    <li>
+                        <a class="nav-link" href="/signup">Signup</a>
+                    </li>
```

Comment:
I added the signup form to its own page to not clutter up the home page. I linked the page to the navbar fragment in this piece of code.